app/views/layouts/application.html.erb

```diff
@@ -9,8 +9,6 @@
     <%= render partial: "layouts/head" %>
     <%= discourse_csrf_tags %>
 
-    <%= render partial: "common/discourse_stylesheet" %>
```

Comment:
Moving these later in the HTML will mean that they **start** loading later. Given they're at the top of the `<body>`, it shouldn't make too much difference. But if we want to move them any later in the `<body>`, or we notice any perf drop, we may want to add some `<link rel="preload"` to the `<head>`

Reply:
Correct :+1: 

Moving them to the top of the `<body>` won't make much difference just based on this PR. However, the goal is to add the splash screen markup before them in the `<body>` tag in a separate PR. 

The order right now is like this

```
<head>
   { stylesheets } ...
</head>
<body>
   { content } ...
</body>
```

If we just add the splash screen to the body tag, it will be blocked by the stylesheets. The goal is something like this

```
<head>
  { stylesheet preload links } ...
</head>
<body>
   { splash } ...
   { stylesheets } ...
   { content } ...
</body>
```

This PR only moves the stylesheets. The splash PR will add the splash markup. We can also have a separate PR to add the preload links though I'm not sure about the amount of improvement those preload links would add (I've not tested that yet)

How does that sound?

Reply:
Sounds good to me :+1: